pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/LuceneRealtimeClusterIntegrationTest.java

```diff
@@ -112,12 +112,12 @@ public void setUp()
 
     // Start the Pinot cluster
     startZk();
-    startController();
-    startBroker();
-    startServer();
 
     // Start Kafka
     startKafka();
+    startController();
```

Comment:
Why do we move all the pinot components after starting kafka?
If there is a reason, could you update the comments above?

Reply:
This follows the same order as we stop them. Also, we should start upstream data sources first.